### ISSUE TITLE
[BUG] Excessive read/write requests while changing the volume level

### ISSUE THREAD

**Issue by spgls** [Moosync/Moosync#324]
**Describe the bug**
Excessive number of file write requests. Adjusting the volume slider causes an EXTREME number of file write requests (Probably the config files)

**To Reproduce**

1. Install Moosync either through Deb package, or through Flatpak
2. Start Moosync and do a initial config
3. Add at least one song to the play queue
4. Attach the tracer to the player process with `sudo strace -e trace=read,write -p <PROCESS ID>` replace `<PROCESS ID>` with the corresponding process ID
5. Change the volume level to see a excessive read/write requests are occurring

**Expected behavior**
Keep the application config in RAM, and save it to the file system when the application exits. Or perform a save every certain period of time (If there are changes), and on exit to save the application state.

**Video proof**
Here a YouTube link with demonstration of issue https://youtu.be/WlOzIKMSbV8

- OS: Linux Mint 21.3 x64
- Sound Server: PulseAudio 15.99.1
- Version: Both Flatpak and Standalone 10.3.2

**Additional context**
For me, changing the volume from 20% to 80% gets around 18 write requests, and twice as much on reads

Interesting fact: It would take about 27778 hours or about 1157 days of uninterrupted volume changing to take out an SSD with a write resource of 200 terabytes

**Comment by Ovenoboyo** [comment]
Yeah, every time you change the volume, it writes preferences back to the config file. Which IG is pretty stupid and I should probably add a debounce on it. Thanks for reporting it though
